test/toc-check.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+require('colors');
+var toc = require('markdown-toc');
+var Diff = require('diff');
+var fs = require("fs");
+var glob = require("glob");
+
+var files = glob.sync("**/*.md", {ignore: ["node_modules/**/*.md"]})
+
+function removeTocHeader(str, ele, arr) {
+  return str.indexOf('Table of Contents') === -1;
+}
+
+function getCurrentToc(str, remove){
+  if(remove === undefined) remove = 0;
+  var startIndex = str.indexOf('Table of Contents') + 19;
```

Comment:
Why 19?

Reply:
It was an easy way to test the files and I forgot to modified it, I already fix it :D